action.yml

```diff
@@ -0,0 +1,124 @@
+name: "Release project"
+inputs:
+  repository:
+    description: 'Repository to release'
+    required: true
+  source-branch:
+    description: 'Source branch to release'
+    required: true
+  gh-runner-token:
+    description: 'Token to clone GitHub repositories'
+    required: true
+  github-token:
+    description: 'Automatic GitHub Workflow token'
+    required: true
+  maven-repository-user:
+    description: 'User to access maven-repository artifacts'
+    required: false
+  maven-repository-token:
+    description: 'Token to access maven-repository artifacts'
+    required: false
+  maven-repository-url:
+    description: 'Url to access maven-repository artifacts'
+    required: false
+  jira-user:
+    description: 'User to access Jira API'
+    required: true
+  jira-token:
+    description: 'Token to access Jira API'
+    required: true
+  quay-api-token:
+    description: 'Application token to access the Quay API'
+    required: true
+  slack-token:
+    description: 'Token to access Slack API'
+    required: true
+  quay-retry-max:
+    description: 'Maximum number of retries to check for the Quay artifact'
+    required: false
+  quay-retry-wait-in-seconds:
+    description: 'Seconds to wait between retries when checking for Quay artifacts'
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout the target repository
+      uses: actions/checkout@v3
+      with:
+        token: ${{ inputs.github-token }}
+        repository: gbtec-ag/${{ inputs.repository }}
+        path: ${{ inputs.repository }}
+        fetch-depth: 0
+
+    - name: Checkout git-repository-runner
+      uses: actions/checkout@v3
+      with: 
+        token: ${{ inputs.gh-runner-token }}
```

Comment:
Why do we need a different token to checkout `git-repository-runner` than the used token to checkout the repository above?

Reply:
GITHUB_TOKEN is auto created by gh actions and scoped the the current repository only.
To checkout other repos, we need appropriate tokens.